src/rcv/swiftnav.c

```diff
@@ -1168,7 +1181,8 @@ static int decode_galnav(raw_t *raw) {
   }
 
   const uint8_t source = puiTmp[156];
-  eph.code = (1 == source) ? 0x2 : 0x5;
+  eph.code = (source == 1) ? (GAL_BROADCAST_ORBIT5_DATA_SOURCE_FNAV_E5A_I | GAL_BROADCAST_ORBIT5_TOC_SISA_E5A) :
```

Comment:
It seems like even with this change, the F/NAV ephemeris won't be selected, because `sel` always equals to 1?

Reply:
Correct: by default, `sel` will be 1 and I/NAV will be used. However there is a function in the RTKLIB API called `satseleph()` which can be used to adjust the contents of the `eph_sel` array.  This is not used by any of the apps in the RTKLIB repo, but it is possible that third party programs might call `satseleph(SYS_GAL, 2)` to use F/NAV instead.